SimplenoteUITests/EmailLogin.swift

```diff
@@ -3,7 +3,10 @@ import XCTest
 class EmailLogin {
 
     class func open() {
+        app.buttons[UID.Button.logIn].waitForIsHittable()
```

Comment:
In some cases, the test might try to tap `Log In` button while the `Save password` prompt is still being removed from the screen after tapping `No` (on this picture, `Login` button is covered by the prompt):

<img width="272" alt="Screenshot 2023-11-02 at 23 04 34" src="https://github.com/Automattic/simplenote-ios/assets/73365754/5e191c76-3891-4e16-9c2f-26665f7c7a28">

Waiting for the button to become hittable solves the issue.